hw2/05-form.js

```diff
@@ -1 +1,52 @@
-// Add your code here
+const form = document.getElementById('form-exercise');
+
+form.addEventListener('submit', handleSubmit);
+
+function handleSubmit(event) {
+  const name = document.getElementById('name');
+  const email = document.getElementById('email');
+  const registration = document.getElementById('registration');
+  const previousClasses = document.getElementById('boxes');
+  const checkboxes = previousClasses.querySelectorAll('input[type="checkbox"]');
+  const anythingElse = document.getElementById('anything-else');
+  const noSubmission = 'no submission';
+
+  if (!name.value && !email.value) {
```

Comment:
Another way you could do input validation for empties is by adding the "required" attribute to the input fields for name and email. With this attribute bootstrap will automatically make the user enter a value before they can submit. it also gives the user a cool top tip message when they try to submit a empty field that is required.

Reply:
Thanks! Added a required attribute to the name